httpx/_api.py

```diff
@@ -49,8 +47,7 @@ def request(
     proxy: ProxyTypes | None = None,
     timeout: TimeoutTypes = DEFAULT_TIMEOUT_CONFIG,
     follow_redirects: bool = False,
-    verify: VerifyTypes = True,
-    cert: CertTypes | None = None,
+    ssl_context: typing.Optional[SSLContext] = None,
```

Comment:
We should be using `typing.Optional[ssl.SSLContext]` here. (Yes it's a bit subtle)